TREnvironmentEditor/EMEditorMapping.cs

```diff
@@ -93,6 +95,10 @@ public void AlternateTextures()
             {
                 OneOf.ForEach(s => s.RemapTextures(AlternativeTextures));
             }
+            if (ConditionalOneOf != null)
+            {
+                ConditionalOneOf.ForEach(s => s.RemapTextures(AlternativeTextures));
```

Comment:
If you're on >= C# 6.0 you can do `ConditionalOneOf?.ForEach` and remove the `if != null`

Reply:
Thanks, I always forget about this.